pkgs/by-name/ni/nixfmt/update.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p cabal2nix curl jq
```

Comment:
Looks like we're also not specifying a nixpkgs revision in our own update script.

If there's no easy way to get the pinned nixpkgs here, should we at least have `-I nixpkgs=.` like many other scripts have?

The downside of using `.` is we're more likely to depend on a package that isn't built by hydra, so perhaps using the user's NIX_PATH is preferable after all.